_content/essays/buddhism-trees-and-forests_dhammika.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Buddhism, Trees, and Forests"
+authors:
+  - dhammika
+drive_links:
+  - "https://drive.google.com/file/d/14Gp_ERzbNG0Ob8USGAUzO7pqx98uFuC1/view?usp=drive_link"
+formats: [pdf]
+course: nature
+tags:
+  - trees
+  - jataka
+booktitle: "Buddhism, Trees, and Forests"
+pages: "3"
```

Comment:
yeah, it's really a 2 page essay, not including the cover. And page counts cannot be in quotes 😅